eslint.config.cjs

```diff
@@ -0,0 +1,33 @@
+const globals = require("globals");
+const js = require("@eslint/js");
+
+const {
+    FlatCompat,
+} = require("@eslint/eslintrc");
+
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+
+module.exports = [{
+    ignores: ["dist/*", "**/testng-reports.js", "**/jquery-1.7.1.min.js"],
+}, ...compat.extends("eslint:recommended"), {
+    languageOptions: {
+        globals: {
+            ...globals.commonjs,
+            ...globals.node,
+            ...globals.jest,
+            Atomics: "readonly",
+            SharedArrayBuffer: "readonly",
+        },
+
+        ecmaVersion: 2018,
+        sourceType: "commonjs",
+    },
+
+    rules: {
+        indent: 2,
+    },
+}];
```

Comment:
add new line

Reply:
Sure, this file was automatically generated by a migration tool.